src/test/libcephfs/test.cc

```diff
@@ -3017,6 +3017,17 @@ TEST(LibCephFS, Readlinkat) {
   target[target_len] = '\0';
   ASSERT_EQ(0, memcmp(target, rel_file_path, target_len));
 
+#if defined(__linux__) && defined(O_PATH)
```

Comment:
this is a special case, might add a comment stating "testing special case with O_NOFOLLOW bits set" or something similar, just for better readability?

Reply:
Fine, updated the comment from LINENO:3021.